src/analysis/souffle/dataflow_graph.dl

```diff
@@ -73,4 +89,21 @@ path(from, to) :- resolvedEdge(_, from, intermediate), path(intermediate, to).
 isAccessPath(x) :- resolvedEdge(_, x, _).
 isAccessPath(y) :- resolvedEdge(_, _, y).
 
+// The 0th predecessor shall be the predecessor with the smallest src ordinal
+// number.
```

Comment:
It took me a moment to figure out that the `ord` is not a user defined thing and that it's the built-in described [here](https://souffle-lang.github.io/types) (I saw this in the souffle docs forever ago but forgot it existed). Maybe it's useful to mention that this is a built-in functor and give the link.